packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.tsx

```diff
@@ -9,18 +9,20 @@ import {
   SceneHeaderText,
   SceneSubHeaderText
 } from 'components/Layout'
+import { ModalName } from 'data/modals/types'
 
+import { Props } from '.'
 import { IntroCardWrapper, Wrapper } from './model'
 
-const OrderMyCardBtn = () => (
+const OrderMyCardBtn = ({ onClick }) => (
   <div style={{ marginTop: '32px' }}>
     <Button
       data-e2e='orderMyCard'
       nature='primary'
       height='48px'
       width='327px'
```

Comment:
Create `styled-component` with proper sizes

Reply:
I used the size I've got from the UX. Which are the proper sizes that you mean?

Reply:
well, the `width` can be `100%` here, right? and the `height` is fine

Reply:
It cannot be width: 100%, this is the mocks the UX did in figma
<img width="903" alt="image" src="https://user-images.githubusercontent.com/97310750/155172880-3d2066ac-cf70-4e22-b647-9cd3bb0a3290.png">